dev/io.openliberty.pages.4.0.internal.factories/src/org/apache/jasper/runtime/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2010 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/**
+ * @version 2.3.0
+ */
+@org.osgi.annotation.versioning.Version("2.3.0")
```

Comment:
Does this seem odd having 2.3.0 here but then exporting in the bnd.bnd at 4.0.0? I know we have the same thing in the pages.3.0.internal.factories bundle. Can you check on this?

Reply:
I think these are import by com.ibm.ws.jsp:
https://github.com/OpenLiberty/open-liberty/blob/ff0156b486c94f355583985fbd5f71859b60504c/dev/com.ibm.ws.jsp/bnd.bnd#L102-L104

I'm not even sure if we need this package info file since we export the package via the bnd (such as in 3.1):
https://github.com/OpenLiberty/open-liberty/blob/ff0156b486c94f355583985fbd5f71859b60504c/dev/io.openliberty.pages.3.1.internal.factories/bnd.bnd#L43

Reply:
Testing a build [here](https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_6n40AFiIEe63MKOTK2E49A) to see if everything passes with it removed.

Reply:
Needed to re-run the build, but all tests passed. I would prefer to remove this from our packages then.